### DISCUSSION TITLE
Commonalities summer break

### DISCUSSION THREAD

**Discussion by shilpa-padgaonkar** [camaraproject/WorkingGroups#261] [Polls]
A quick check to see if we would like to cancel a call or two during summer break due to summer holidays related absences.
Please vote for the meeting slots you would like to skip.

**Comment by bigludo7** [comment]
@shilpa radio button doesn't fit with French vacations :)
I will enjoy National Day July 14th and then OoO both July 17th & Aug 11th

So back Aug 24th for me :)

**Comment by shilpa-padgaonkar** [comment]
We will skip 11th Aug commonalities call.
